wagtail/admin/templates/wagtailadmin/shared/side_panels/preview.html

```diff
@@ -54,7 +54,9 @@ <h3 class="preview-panel__error-title">
         </div>
 
         <div class="preview-panel__wrapper">
-            <iframe loading="lazy" title="{% trans 'Preview' %}" class="preview-panel__iframe" data-preview-iframe aria-describedby="preview-panel-error-banner"></iframe>
+            <iframe loading="lazy" title="{% trans 'Preview' %}" class="preview-panel__iframe" data-preview-iframe aria-describedby="preview-panel-error-banner">
+                <div/>
```

Comment:
```suggestion
                <div{# Add placeholder element to support styling content when iframe has loaded #}/>
```

This probably needs a comment to explain why it's here.